### PR TITLE
Update the docs index page

### PR DESCRIPTION
**Pull request by djw8605**
Minor changes:
- Fix location of search bar
- Make note of 3.4 release, and a more general statement to update between them.

Major Changes:
- Add an ordered list of items to install an OSG site.
- Change order so that installation of CE is before WN.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -35,7 +35,15 @@ For smaller sites (less than 50 worker nodes), the most common way to add a site
 Add OSG Software
 ----------------
 
-If necessary, provision all OSG hosts that are in your site plan and that do not exist yet.
+If necessary, provision all OSG hosts that are in your site plan and that do not exist yet.  The general steps to installing an OSG site are:
```
Comment (brianhlin) [inline docs/index.md]: It doesn't seem like we need `and` in the first sentence

File: docs/index.md [hunk 2]
```diff
@@ -45,7 +53,7 @@ If necessary, provision all OSG hosts that are in your site plan and that do not
 -   [Security information for OSG signed RPMs](release/signing)
 -   [Using Yum and RPM](release/yum-basics)
 -   [Install the OSG repositories](common/yum)
--   [OSG Software release series](release/release_series) - look here to upgrade from OSG 3.1 to OSG 3.2 or from OSG 3.2 to OSG 3.3
+-   [OSG Software release series](release/release_series) - look here to upgrade between releases 3.1 to 3.4.
```
Comment (brianhlin) [inline docs/index.md]: This sounds like there are only 3.1 -> 3.4 instructions. How about `..between releases 3.1, 3.2, or 3.3 to 3.4`?

File: docs/index.md [hunk 3]
```diff
@@ -35,7 +35,15 @@ For smaller sites (less than 50 worker nodes), the most common way to add a site
 Add OSG Software
 ----------------
 
-If necessary, provision all OSG hosts that are in your site plan and that do not exist yet.
+If necessary, provision all OSG hosts that are in your site plan and that do not exist yet.  The general steps to installing an OSG site are:
+
+1. [Retrive a Host Certificate](#installing-and-managing-certificates-for-site-security) for your CE.
```
Comment (brianhlin) [inline docs/index.md]: Drop this step, it's mentioned in the CE install doc

File: docs/index.md [hunk 4]
```diff
@@ -35,7 +35,15 @@ For smaller sites (less than 50 worker nodes), the most common way to add a site
...
+1. Install [OSG Yum Repos](release/yum-basics) and the [Compute Element software](#installing-and-configuring-the-compute-element) on your CE host
+1. Install [OSG Yum Repos](release/yum-basics) and the [Worker Node client](#adding-osg-software-to-worker-nodes) on your worker nodes.
```
Comment (brianhlin) [inline docs/index.md]: They don't necessarily need the yum repos on the WNs if they're doing tarball installs

**Comment by djw8605** [comment]
Updated @brianhlin

**Pull request merged**